PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,11 +26,15 @@ Consider the following testing approaches:
 
 ## How will you know that this is working after deployment?
 
-**REQUIRED**
+**Reminder that you should only move your Jira ticket to "Done" once you've verified that it's been deployed and is functoining as expected in production**
```

Comment:
```suggestion
**Reminder that you should only move your Jira ticket to "Done" once you've verified that it's been deployed and is functioning as expected in production**
```

Reply:
I think the "and is functioning as expected in production" part is controversial here, though since I don't consider the PR template to define policy, I'm also fine simply ignoring it when appropriate.

Unless there are dependencies on other systems or a flaky CI process in play, I don't habitually ensure that, after terraform has completed and all the AWS APIs returned successfully, that AWS processed those updates correctly. So my "how will you verify this is working after deployment" for straightforward code changes is generally "I'll monitor the deployment"